precompiles/utils/src/data/mod.rs

```diff
@@ -431,6 +431,49 @@ impl EvmData for U256 {
 	}
 }
 
+impl<T> EvmData for Option<T>
```

Comment:
Options don't exist in Solidity and this format doesn't match the layout Solidity uses.